docs/docs/guide/google-tag-manager-and-growthbook.mdx

```diff
@@ -0,0 +1,348 @@
+
+The second strategy does not incur as much of a page render speed tax - the flicker is much less noticeable. However, you’ll need to hard-code all of your experiments and their rulesets, as well as any feature flag rules and overrides, into the GTM tag’s source code. Each time you need to change your experiments or phases, you’ll need to modify the code snippet and publish a new GTM version. For some, this may impose a significant burden; for others, the render speed improvement may be worth the tax.
+
+If neither of these strategies seem ideal, we’d encourage you to pursue a traditional GrowthBook implementation without GTM, preferably with back-end or hybrid feature flag evaluation.
+
+## Basic setup
+
+Regardless of which of the above strategies you’ve chosen, in order to implement GrowthBook AB Tests you’ll first need to inject the GrowthBook Javascript SDK using a GTM Tag. You’ll also need to pass some basic information along to the SDK in order to use it.
+
+### Creating a GTM tag for the GrowthBook SDK
+
+First, create a new tag in your desired workspace. We will choose “Custom HTML” as the tag type. We can give it the name “GrowthBook SDK” or similar. Also, be sure to set the firing triggers to target the specific pages where we need to instrument our feature changes and experiments (or just choose “All Pages”).
+
+Next, paste in the script below to load the GrowthBook JavaScript SDK; then save the tag:
+
+```js
+<script id="growthbook-sdk" src="https://cdn.jsdelivr.net/npm/@growthbook/growthbook/dist/bundles/index.min.js" defer></script>
+```
+
+To publish the SDK tag, submit our workspace changes (the blue “Submit” button on the top of the GTM application), Then ensure “Publish and Create version” is selected and click the blue “Publish” button – or use whichever GTM release strategy you are already using.
+
+## Initializing the SDK
+
+To actually use the GrowthBook SDK on our pages to control on-page features with AB tests, we’ll need to create another code snippet and load it through GTM. While you can certainly add the snippet to your existing tag (the GrowthBook SDK tag we created above), you may find it cleaner to add another Custom HTML tag and give it a name such as “GrowthBook Implementation” Be sure to set the firing triggers in the same way as you did above.
+
+Insert the following code into your tag:
```

Comment:
Do we have to call out that they should replace the clickKey and other attributes? What about the tracking callback? - ah, read lower down - so perhaps we should let folks know we are going into detail and updating this snippet.